src/apps/AppCenter/DiscoverApps/DiscoverApps.js

```diff
@@ -1,121 +1,202 @@
+        <Link
+          href="https://hack.aragon.org/docs/cli-dao-commands#dao-install"
+          target="_blank"
+        >
+          CLI guide
+        </Link>{' '}
+        and learn{' '}
+        <Link href="" target="_blank">
+          how to create apps yourself.
+        </Link>
+        You can also preview some of the apps being developed.
+      </Info>
+      <CardLayout columnWidthMin={columnWidthMin} rowHeight={rowHeight}>
+        {appsInDevelopment.map((app, i) => (
+          <AppCard key={i} app={app} compactMode={compactMode} />
+        ))}
+      </CardLayout>
+    </React.Fragment>
+  )
+})
 
-const AppsGrid = styled.div`
-  display: grid;
-  grid-auto-flow: row;
-  grid-gap: 25px;
-  justify-items: start;
-  grid-template-columns: 1fr;
-  ${breakpoint(
-    'medium',
-    `
-      grid-template-columns: repeat(auto-fill, 224px);
-    `
-  )};
-`
+function AppCard({ app, compactMode, onOpen, ...props }) {
+  const theme = useTheme()
+  const { link, icon, name, status, description } = app
+  const handleClick = useCallback(() => {
+    if (link) {
+      window.open(link, '_blank', 'noopener')
+    }
+  }, [link])
+
+  return (
+    <Card onClick={handleClick}>
+      <Main compactMode={compactMode}>
+        <StyledIconExternal
+          compactMode={compactMode}
+          theme={theme}
+          link={link}
+        />
+        <Icon compactMode={compactMode}>
+          <AppIcon size={9 * GU} src={icon} radius={12} />
+        </Icon>
+        <Name compactMode={compactMode}>{name}</Name>
+        <TagWrapper compactMode={compactMode} link={link}>
+          <Tag mode="indicator">{status}</Tag>
+        </TagWrapper>
+        <Description theme={theme} compactMode={compactMode}>
+          {description}
+        </Description>
+      </Main>
+    </Card>
+  )
+}
 
-const Main = styled(Card).attrs({ width: '100%', height: '288px' })`
+const Main = styled.section`
   ${unselectable};
   position: relative;
   overflow: hidden;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  padding-top: 25px;
+  height: 100%;
+  width: 100%;
+  white-space: initial;
+
+  ${({ compactMode }) =>
+    compactMode
+      ? `
+          display: grid;
+          grid-template-columns: auto 1fr auto;
+          grid-template-rows: auto auto auto;
+          grid-template-areas:
+            "empty topright"
+            "icon title"
+            "icon description";
+          padding: ${1.5 * GU}px ${1.5 * GU}px ${4 * GU}px ${3 * GU}px;
+        `
+      : `
+          display: flex;
+          flex-direction: column;
+          align-items: center;
+          padding-top: ${5 * GU}px;
+        `}
 `
 
 const Icon = styled.div`
-  height: 64px;
-  margin-bottom: 5px;
-  img {
-    display: block;
-  }
+  height: ${9 * GU}px;
+  width: ${9 * GU}px;
+  ${({ compactMode }) =>
+    compactMode
+      ? `
+        grid-area: icon;
+        margin-right: ${1.5 * GU}px;
```

Comment:
We should add `align-self: center` here since it's not centered sometimes:

<img width="350" alt="Screen Shot 2019-08-26 at 7 45 21 PM" src="https://user-images.githubusercontent.com/4166642/63710695-1192c300-c83a-11e9-9dd0-ba2151f12df1.png">